Source/WebCore/Modules/webxr/WebXRSession.cpp

```diff
@@ -306,16 +309,18 @@ IntSize WebXRSession::nativeWebGLFramebufferResolution() const
 // https://immersive-web.github.io/webxr/#recommended-webgl-framebuffer-resolution
 IntSize WebXRSession::recommendedWebGLFramebufferResolution() const
 {
-    ASSERT(m_device);
-    return m_device->recommendedResolution(m_mode);
+    auto device = m_device.get();
+    ASSERT(device);
```

Comment:
Could return an empty IntSize if it's null.

Reply:
Will fix.